clib/hMap.ml

```diff
@@ -249,14 +249,6 @@ struct
       let m = Map.singleton k x in
       Int.Map.add h m s
 
-  (* when Coq requires OCaml 4.06 or later, the module type
-     CSig.MapS may include the signature of OCaml's "update",
-     requiring an implementation here, which could be just:
-
-       let update k f s = assert false (* not implemented *)
```

Comment:
To any other reviewer who may be wondering: it already got implemented around line 426